angular/projects/admin/src/app/admin/settings/settings.component.ts

```diff
@@ -99,6 +134,8 @@ export class SettingsComponent implements OnInit, OnDestroy {
     return this.siteSettingsService.save({
       title: formData.title,
       theme: formData.theme,
+      defaultLanguage: formData.defaultLanguage,
+      languages: this.languages.map(lang => lang.id),
```

Comment:
Sort the languages array so that the default language is on index `0`. Because later [in the modal dialog you are assuming that default language is index `0` by preventing deletion of first entry](https://github.com/oddbit/tanam/pull/51/files#diff-a8e8c431e877bb75eb757e9acf7c9530R46).

![image](https://user-images.githubusercontent.com/2051607/53707901-e7a0ab00-3e6b-11e9-9247-b92f1e6a98df.png)

Reply:
Resolve on this commit